docs/openstack/create-and-connect-to-the-VM/launch-a-VM.md

```diff
@@ -93,67 +98,79 @@ you may choose any available images.
     storage.
 
-- **Flavor** tab: Specify the size of the instance to launch. Choose `cpu-su.4`
-from the 'Flavor' tab by clicking on the "+" icon.
+### **Flavor** Tab
+
+Specify the size of the instance to launch. Choose `cpu-su.4` from the 'Flavor'
+tab by clicking on the "+" icon.
 
 !!! question "Important Note"
     In NERC OpenStack, flavors define the compute, memory, and storage
     capacity of nova computing instances. In other words, a flavor is an
     available hardware configuration for a server.
 
-    Some of the flavors will not be available for your use as per your resource **Quota**
-    limits and will be shown as below:
+    Some of the flavors will not be available for your use as per your resource
+    **Quota** limits and will be shown as below:
 
     ![Flavor Not Avaliable due to Your Quota](images/flavor-not-available-due-to-quota.png)
 
-    **More details about available flavors can be found [here](flavors.md) and
-    how to change request the current allocation quota attributes can be found [here](../../get-started/get-an-allocation.md#request-change-resource-allocation-attributes-for-openstack-project).**
+    More details about available flavors can be found [**here**](flavors.md) and
+    how to change request the current allocation quota attributes can be found
+    [**here**](../../get-started/allocation/allocation-change-request.md#request-change-resource-allocation-attributes-for-openstack-project).
 
 After choosing `cpu-su.4`, you should see it moved up to "Allocated".
 
 ![VM Launch Instance Flavor](images/launch_flavor.png)
 
-- **Networks:** tab: Make sure the Default Network that is created by default is
-moved up to "Allocated". If not, you can click on the "+" icon in "Available".
+!!! info "Storage and Volume"
+    - System disks are the first disk based on the flavor disk space and are
+    generally used to store the operating system created from an image when the
+    virtual machine is booted.
+    - [Volumes](../persistent-storage/volumes.md) are
+    persistent virtualized block devices independent of any particular instance.
+    Volumes may be attached to a single instance at a time, but may be detached
+    or reattached to a different instance while retaining all data, much like a
+    USB drive. The size of the volume can be selected when it is created within
+    the storage quota limits for the particular resource allocation.
 
-![VM Launch Instance Networks](images/launch_networks.png)
+### **Networks** Tab
 
-- **Security Groups:** tab: Make sure to add the security group where you
-enabled SSH. To add an SSH security group first, see [here](../access-and-security/security-groups.md).
+Make sure the Default Network that is created by default is moved up to "Allocated".
+If not, you can click on the "+" icon in "Available".
 
-![VM Launch Instance Security Groups](images/launch_security_groups.png)
+![VM Launch Instance Networks](images/launch_networks.png)
```

Comment:
no image is displayed in the Network section in the preview.

Reply:
It loads slowly

Reply:
now I see it is a display issue of comparing Markdown to the NERC html display. looks good.

Reply:
So, if we have https://squidfunk.github.io/mkdocs-material/reference/admonitions/ in markdown this will not render the images.